io.github.daviddesimone.opencloudsaves.yml

```diff
@@ -0,0 +1,74 @@
+app-id: io.github.daviddesimone.opencloudsaves
+base: io.elementary.BaseApp
+base-version: juno-21.08
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: bootstrap-opencloudsave.sh
+cleanup:
+  - /app/lib/sdk/golang
+finish-args:
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--filesystem=host'
+  - '--socket=wayland'
+  - '--device=dri'
+  - '--share=network'
+build-options:
+  env:
+    - GOBIN=/app/bin
+    - GOROOT=/app/lib/sdk/golang
+modules:
+  - name: golang
+    buildsystem: simple
+    sources:
+      - type: archive
+        only-arches:
+          - aarch64
+        url: https://go.dev/dl/go1.19.5.linux-arm64.tar.gz
+        sha256: fc0aa29c933cec8d76f5435d859aaf42249aa08c74eb2d154689ae44c08d23b3
+      - type: archive
+        only-arches:
+          - x86_64
+        url: https://go.dev/dl/go1.19.5.linux-amd64.tar.gz
+        sha256: 36519702ae2fd573c9869461990ae550c8c0d955cd28d2827a6b159fda81ff95
+    build-commands:
+      - install -d /app/lib/sdk/golang
+      - cp -rpv * /app/lib/sdk/golang/
+  - name: libdevhelp
+    buildsystem: meson
+    config-opts:
+        - "-Dflatpak_build=true"
+    sources:
+      - type: archive
+        url: https://gitlab.gnome.org/GNOME/devhelp/-/archive/41.2/devhelp-41.2.tar.gz
+        sha256: 2798546427a42484b61733f13d8532f31e5aa0bdff4a22aa1dc8e3085358fc87
+  - name: opencloudsave
+    buildsystem: simple    
+    build-commands:
+      - $GOROOT/bin/go build -trimpath -o opencloudsave
+      - install -D opencloudsave /app/bin/opencloudsave
+      - install -D ./build/linux/bootstrap-opencloudsave.sh /app/bin/bootstrap-opencloudsave.sh
+      - mkdir -p /app/share/applications/
+      - install -D ./build/linux/icon/io.github.daviddesimone.opencloudsaves.512.png /app/share/icons/hicolor/512x512/apps/io.github.daviddesimone.opencloudsaves.png
+      - install -D ./build/linux/icon/io.github.daviddesimone.opencloudsaves.256.png /app/share/icons/hicolor/256x256/apps/io.github.daviddesimone.opencloudsaves.png
+      - install -D ./build/linux/icon/io.github.daviddesimone.opencloudsaves.128.png /app/share/icons/hicolor/128x128/apps/io.github.daviddesimone.opencloudsaves.png
+      - install -D ./build/linux/icon/io.github.daviddesimone.opencloudsaves.64.png /app/share/icons/hicolor/64x64/apps/io.github.daviddesimone.opencloudsaves.png
+      - install -D ./build/linux/icon/io.github.daviddesimone.opencloudsaves.32.png /app/share/icons/hicolor/32x32/apps/io.github.daviddesimone.opencloudsaves.png
```

Comment:
Unless they're used internally by the app, just 512x512 would do the trick.

Reply:
Corrected.